xwebrtc/client.go

```diff
@@ -0,0 +1,32 @@
+package xwebrtc
+
+import (
+	"cdr.dev/slog"
+	"context"
+	"golang.org/x/net/proxy"
+	"golang.org/x/xerrors"
+	"net/url"
+)
+
+type WorkspaceClient struct {
+	log            slog.Logger
+	brokerAddr     *url.URL
+	token          string
+}
+
+func NewWorkspaceClient(log slog.Logger, brokerAddr *url.URL, token string) *WorkspaceClient {
+	return &WorkspaceClient{
+		log: log,
+		brokerAddr: brokerAddr,
+		token: token,
+	}
+}
```

Comment:
Does this need to exist anymore? We could just add these args to `DialWorkspace` imo

Reply:
gone